app/signup/template.hbs

```diff
@@ -68,13 +70,15 @@
         <h2>
           Aptible is a deployment platform built to automate HIPAA compliance
         </h2>
-
         <p>
-          Run the applications and databases of your choice. Let us manage servers, network topology, security, backups, encryption,
+          We run the entire backend you need to securely host and deliver your apps: web servers, app servers, databases, load balancers, network security, backups, encryption, permissions - everything.
+        </p>
+        <p>
+          And because we run your ops, our compliance engines can help you implement an entire HIPAA compliance program at a fraction of the traditional cost.
         </p>
 
         <p>
-          Aptible will work great with your app’s language or framework.  Check out our getting started guides for help deploying your app.
+          Aptible works great with all languages and frameworks. <a href="https://support.aptible.com/quickstart" target="_blank">Check out our getting started guides</a> for help deploying your app.
```

Comment:
Hm, would like to see this as:

```hbs
{{#link-to-aptible app="support" path="quickstart" target="_blank"}}
  Check out our getting started guides
{{/link-to-aptible}}
```

Thought I think `target` support for the `link-to-aptible` component would be new.